packages/php-psr/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jbboehr/php-psr
+TERMUX_PKG_DESCRIPTION="PHP extension providing the accepted PSR interfaces"
+TERMUX_PKG_LICENSE="Simplified BSD"
+TERMUX_PKG_LICENSE_FILE=LICENSE.md
```

Comment:
Not required

Reply:
`TERMUX_PKG_LICENSE_FILE ` is required, without it the build scripts return an `ERROR: No LICENSE file was installed for php-psr`

Reply:
It should not be required, LICENSE.md is in the list of searched file names: https://github.com/termux/termux-packages/blob/5028aaca6ad098bba95775d64a7ccd294ed9412a/scripts/build/termux_step_install_license.sh#L38

Reply:
My bad, Simplified BSD is in the list of licenses with copyright information. I fixed that in https://github.com/termux/termux-packages/commit/73bc3b19ca92023ca1244f8db83d7699c5395d2a, so if you rebase this onto master you can drop that line

Reply:
I test it again, the error is not occur, and I removed the `TERMUX_PKG_LICENSE_FILE`

Reply:
> My bad, Simplified BSD is in the list of licenses with copyright information. I fixed that in [73bc3b1](https://github.com/termux/termux-packages/commit/73bc3b19ca92023ca1244f8db83d7699c5395d2a), so if you rebase this onto master you can drop that line

done

Reply:
> My bad, Simplified BSD is in the list of licenses with copyright information. I fixed that in [73bc3b1](https://github.com/termux/termux-packages/commit/73bc3b19ca92023ca1244f8db83d7699c5395d2a), so if you rebase this onto master you can drop that line

As @thunder-coding said: it should be `BSD Simplified` , not `Simplified BSD`